source/_integrations/livisi.markdown

```diff
@@ -2,14 +2,16 @@
 title: LIVISI SmartHome
 description: Access and control your LIVISI Smart Home Controller (SHC) and its connected RWE/innogy devices.
 ha_category:
+  - BinarySensor
```

Comment:
This category does not exist, I guess you meant `Binary Sensor` (with a space).

Reply:
I changed this, thank you.